src/components/SearchByIDA/IdaEntry/index.tsx

```diff
@@ -0,0 +1,260 @@
+    const style: Record<string, { background?: string }> = options[
+      this.props.entry
+    ]
+      ? {
+          background: getTrackColor(
+            { accession: this.props.entry, source_database: '' },
+            EntryColorMode.ACCESSION,
+          ),
+        }
+      : {};
+    return (
+      <div
+        className={css('ida-entry')}
+        draggable={this.state.draggable}
+        onDragStart={this._handleStartDragging}
+        onDragEnd={this._handleEndDragging}
+        onDragCapture={this._handleDragging}
+        ref={this.container}
+        style={style}
+      >
+        <Select
+          options={Object.values(options).map(({ accession, name }) => ({
+            value: accession,
+            label: name,
+          }))}
+          isLoading={
+            this.state.loadingInterProOptions || this.state.loadingPfamOptions
+          }
+          onInputChange={this._handleOnInputChange}
+          onChange={this._handleOnChange}
+          className={css('react-select-container')}
+          value={{
+            value: entry,
+            label: options?.[entry]?.name,
+          }}
+          styles={{
+            menuList: (provided: CSSObjectWithLabel) =>
+              ({
+                ...provided,
+                background: 'white',
+              }) as CSSObjectWithLabel,
+            menu: (provided: CSSObjectWithLabel) =>
+              ({
+                ...provided,
+                top: undefined,
+                width: 'var(--entry-width)',
+                marginTop: 0,
+              }) as CSSObjectWithLabel,
+            control: (provided: CSSObjectWithLabel) =>
+              ({
+                ...provided,
+                background: 'transparent',
+                border: 0,
+                boxShadow: undefined,
+              }) as CSSObjectWithLabel,
+            input: (provided: CSSObjectWithLabel) =>
+              ({
+                ...provided,
+                color: 'white',
+              }) as CSSObjectWithLabel,
+          }}
+          formatOptionLabel={({ value, label }, { context }) => {
+            return (
+              <div
+                style={{
+                  color: context === 'menu' ? 'black' : 'white',
+                  cursor: context === 'menu' ? 'pointer' : 'text',
+                }}
+                className={css('react-select-labels')}
+                key={value}
+              >
+                <div className={css('react-select-label-header')}>{value}</div>
+                {label && (
+                  <div className={css('react-select-label-body')}>{label}</div>
+                )}
+              </div>
+            );
+          }}
+        />
+        {draggable && (
+          <button
+            className={css('drag', { nodata: !options[entry] })}
+            onMouseEnter={() => this.setState({ draggable: true })}
+            onMouseLeave={() => this.setState({ draggable: false })}
+          >
+            |||
```

Comment:
Use the vertical ellipsis icon instead of text to indicate the grab handle:

```suggestion
            <i className={css('icon', 'icon-common', 'icon-ellipsis-v')}></i>
```